htdocs/install/mysql/tables/llx_c_civility.sql

```diff
@@ -19,7 +19,7 @@
 
 create table llx_c_civility
 (
-  rowid       integer    PRIMARY KEY,
+  rowid       integer AUTO_INCREMENT PRIMARY KEY NOT NULL
```

Comment:
If we do so, we should also change into migration script. According to sample we should add this into migration script

-- To make pk to be auto increment (mysql):    
--VMYSQL4.3 ALTER TABLE llx_c_civility CHANGE COLUMN rowid rowid INTEGER NOT NULL AUTO_INCREMENT;

-- To make pk to be auto increment (postgres):
--VPGSQL8.2 CREATE SEQUENCE llx_c_civility OWNED BY llx_c_civility.rowid;
--VPGSQL8.2 ALTER TABLE llx_c_civility ADD PRIMARY KEY (rowid);
--VPGSQL8.2 ALTER TABLE llx_c_civility ALTER COLUMN rowid SET DEFAULT nextval('llx_c_civility_rowid_seq');
--VPGSQL8.2 SELECT setval('llx_c_civility_rowid_seq', MAX(rowid)) FROM llx_c_civility;

Reply:
I did not test on pgsql however

Reply:
Tested on pgsql with this, it's ok

--VPGSQL8.2 CREATE SEQUENCE llx_c_civility_rowid_seq OWNED BY llx_c_civility.rowid;
--VPGSQL8.2 ALTER TABLE llx_c_civility ALTER COLUMN rowid SET DEFAULT nextval('llx_c_civility_rowid_seq');
--VPGSQL8.2 SELECT setval('llx_c_civility_rowid_seq', MAX(rowid)) FROM llx_c_civility;